platform/viewer/.webpack/writePluginImportsFile.js

```diff
@@ -36,56 +36,80 @@ function constructLines(input, categoryName) {
 function getFormattedImportBlock(importLines) {
   let content = '';
   // Imports
-  importLines.forEach(importLine => {
+  importLines.forEach((importLine) => {
     content += importLine;
   });
 
   return content;
 }
 
 function getFormattedWindowBlock(addToWindowLines) {
-  let content = "const extensions = [];\n" +
-    "const modes = [];\n" +
-    "const modesFactory = [];\n" +
-    "window.extensions = extensions;\n" +
-    "window.modes = modes;\n\n";
-
-  addToWindowLines.forEach(addToWindowLine => {
+  let content =
+    'const extensions = [];\n' +
+    'const modes = [];\n' +
+    'const modesFactory = [];\n' +
+    'window.extensions = extensions;\n' +
+    'window.modes = modes;\n\n';
+
+  addToWindowLines.forEach((addToWindowLine) => {
     content += addToWindowLine;
   });
 
   return content;
 }
 
 function getRuntimeLoadModesExtensions() {
-  return "\n\n// Add a dynamic runtime loader\n" +
-    "export default async () => {\n" +
-    " for(const modeFactory of modesFactory) {\n" +
-    "  const newModes = await modeFactory(modes,extensions);\n" +
-    "  newModes.forEach(newMode => modes.push(newMode));\n" +
-    "}\n}\n";
+  return (
+    '\n\n// Add a dynamic runtime loader\n' +
+    'export default async () => {\n' +
+    ' for(const modeFactory of modesFactory) {\n' +
+    '  const newModes = await modeFactory(modes,extensions);\n' +
+    '  newModes.forEach(newMode => modes.push(newMode));\n' +
+    '}\n}\n'
+  );
 }
 
-const createCopyPluginFromExtensions = (SRC_DIR, DIST_DIR, plugins) => {
-
-  return plugins.map(plugin => {
-    const from = `${SRC_DIR}/../node_modules/${plugin.packageName}/public/`;
-    const exists = fs.existsSync(from);
-    return exists ? {
-      from,
-      to: DIST_DIR,
-      toType: 'dir',
-    } : undefined;
-  }
-  ).filter(x => !!x);
-}
+const createCopyPluginPublicToDist = (SRC_DIR, DIST_DIR, plugins) => {
+  return plugins
+    .map((plugin) => {
+      const from = `${SRC_DIR}/../../../node_modules/${plugin.packageName}/public/`;
```

Comment:
The SRC_DIR/../node_modules is used for link, while the SRC_DIR/../../../node_modules is used for package modules.  Ugly that it is different, but yarn is buggy.